test/sql/visualizer/test_visualizer.test

```diff
@@ -0,0 +1,47 @@
+# name: test/sql/visualizer/test_visualizer.test
+# description: Profiler Visualizer
+# group: [visualizer]
+
+require visualizer
+
+statement ok
+PRAGMA enable_profiling='json'
+
+statement ok
+Pragma visualize_last_profiling_output ='__TEST_DIR__/test.html'
+
+statement ok
+PRAGMA profiling_output='__TEST_DIR__/first.json'
+
+statement ok
+PRAGMA profiling_mode = detailed
```

Comment:
I see that there are many `PRAGMA` needed before being able to visualize the profiling. I wonder if it would be better if `PRAGMA visualize_last_profiling_output` automatically sets `PRAGMA enable_profiling='json' and `PRAGMA profiling_mode = detailed`.

What happens now if a user does `PRAGMA visualize_last_profiling_output` without setting `PRAGMA enable_profiling='json' and `PRAGMA profiling_mode = detailed`? Does it crash? Does it throw an error? If it throws an error, this should be tested, e.g. with `statement error`. In general, it is a good idea to test these pragma's with different input orders and different values.

Reply:
The visualize_last_profiling_output is not dependant on both of the pragmas you mentioned. PRAGMA enable_profiling='json' is only needed when someone wants to save profiler-output into a JSON file. The visualize_last_profiling_output will automatically generate the profiler-output as a JSON string and inject it into the HTML file.